.travis.yml

```diff
@@ -0,0 +1,23 @@
+# More details on how to configure the Travis build
+# https://docs.travis-ci.com/user/customizing-the-build/
+
+language: java
```

Comment:
Should this be Scala? Or Scala + Java?

Reply:
Good point! Java is a good start and it is required for Gradle / Scala. Once we start testing with Travis CI we will discover if this needs to be configured differently.

Reply:
This [guide](https://www.scala-sbt.org/0.13/docs/Travis-CI-with-sbt.html) has the following setting.

```
language: scala

scala:
   - 2.11.11
   - 2.12.4
```